### DISCUSSION TITLE
Option to change favicon icon, page title and admin logo

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo_admin#201] [Ideas]
Currently, in the piccolo-admin we can change site_name , but not the favicon (Browser Tab Icon), page title (Browser Tab title) and admin logo(Picture left to the site_name). 
Please provide an option to customise these too.
Thank you

**Comment by sinisaos** [comment]
@sumitsharansatsangi Can the page title be the same as the custom `site_name` or does it have to be different?

We can create a new folder called `icons` in the public Vue folder. Then you (or another user) just need to add your `favicon.ico` and your `logo image` to Piccolo Admin dist folder. Is this okay?

Usage would be 
```python
create_admin(
    ...
    site_name="your_custom_site_name",
    page_title="your_custom_page_title",
    logo_name="your_logo_filename.jpg", # or any other extension supported by <img src>
    logo_height=50, # or any other integer
    logo_width=50, # or any other integer
),
```
Result is:

![custom](https://user-images.githubusercontent.com/30960668/184473778-c5a648f2-135a-4395-b016-b49e3dfef8b9.png)

@dantownsend What do you think about this?

**Reply by sumitsharansatsangi** [reply]
Page title can be same as the `site_name`. There is no need of adding a different parameter.

Yes, creating a folder called icons in the public view folder and adding favicon.ico  will be fine.

If you don't want to change the logo, please don't,  it's fine to be there, no issue. 

Is just  adding icons folder and page title requires 'npm run build', not know much about javascript / typescript .

**Reply by sumitsharansatsangi** [reply]
Yeah, received your last edit just, It is okay.

**Reply by sinisaos** [reply]
Ok. If @dantownsend says it's ok, I can do a PR for this.

**Reply by dantownsend** [reply]
I think we should definitely make the HTML page title reflect the `site_name` parameter.

Customising the icon is a bit trickier. We could experiment with that in a separate PR.

**Reply by sinisaos** [reply]
I'll do a PR later today so we'll see what we can use and what not.

**Comment by dantownsend** [comment]
In the next release it will make the HTML title match the `site_name`.

I couldn't make the custom logo stuff efficient. The app re-renders the nav bar quite frequently, which meant it pulled the logo from the server all the time. If we refactor the Vue components in the future, it might be possible to add the custom logo feature back.
